src/settings.h

```diff
@@ -1,42 +1,57 @@
-// SPDX-FileCopyrightText: 2023 ThingPulse Ltd., https://thingpulse.com
-// SPDX-License-Identifier: MIT
+/*-------------------------------------------------------------------------------------------------
+**
+** settings.h
+**
+**    Main constants used by the application, including versioning,
+**    compile-time info, time zone, display rotation, and system behavior
+**    for the ESP32 Spotify Remote.
+**
+** SPDX-FileCopyrightText: 2025 ThingPulse Ltd., https://thingpulse.com
+** SPDX-License-Identifier: MIT
+**
+** ------------------------------------------------------------------------------------------------
+** Change Log:
+**    2024-12-26 - Electric Diversions - Copied from ThingPulse Spotify Remote.
+**    2024-12-27 - Electric Diversions - Removed blocking declarations and dead code.
+** ------------------------------------------------------------------------------------------------
+*/
 
 #pragma once
+#include "compile_time.h"
 
 // ****************************************************************************
 // User settings
 // ****************************************************************************
-// WiFi
-const char *SSID = "";
-const char *WIFI_PWD = "";
 
-// timezone Europe/Zurich as per https://github.com/nayarsystems/posix_tz_db/blob/master/zones.csv
-#define TIMEZONE "CET-1CEST,M3.5.0,M10.5.0/3"
+/*
+** ===================================================================
+** Version and Name Information
+** ===================================================================
+*/
+constexpr const char* APP_NAME     = "ESP32 Spotify Remote";
+constexpr const char* VERSION      = "2.0.0";
+constexpr const char* COMPILE_TIME = SC_COMPILE_TIME;
 
-// Spotify settings
-const char *SPOTIFY_CLIENT_ID = "";
-const char *SPOTIFY_CLIENT_SECRET = "";
-// Use http://<value-configured-here>.local/callback/ as the redirect URI for the app on Spotify.
-// Hence, the default URI is http://tp-spotify.local/callback/.
-// If you change the value here, you need to modify the redirect URI on Spotify as well.
-#define SPOTIFY_ESPOTIFIER_NODE_NAME "tp-spotify"
+/*
+** ===================================================================
+** Time Zone to use
+** ===================================================================
+**
+** timezone Europe/Zurich as per 
+** https://github.com/nayarsystems/posix_tz_db/blob/master/zones.csv
+*/
+#define TIMEZONE "CST6CDT,M3.2.0,M11.1.0"
```

Comment:
You changed the timezone but not the corresponding comment ("timezone Europe/Zurich as per...").

```
20:46:53.184 > [ 13836][I][util.cpp:58] initTime(): UTC time: 2025-06-09 18:46:53.
20:46:53.190 > [ 13842][I][util.cpp:97] setTimezone(): Setting timezone to 'CST6CDT,M3.2.0,M11.1.0'.
20:46:53.193 > [ 13850][I][SCLogger.cpp:94] logMessage(): [General] Current local time: 2025-06-09 13:46:53 [src/main.cpp:272][I] (loopTask)
```

Also, I feel it would make sense to reintroduce the `#ifdef DATE_TIME_FORMAT_US`-section in this file and use the format specifiers (→ `HomeView::handleClock()`). The time is currently displayed in US format.